flottform/forms/src/theme/defaultTheme.ts

```diff
@@ -191,108 +48,189 @@ const createLinkAndQrCode = (qrCode: string, link: string) => {
 	};
 };
 const defaultThemeForAnyInput =
-	(
-		options: {
-			id?: string;
-			flottformRootElement?: HTMLElement;
-			additionalItemClasses?: string;
-		} = {}
-	) =>
+	(inputField?: HTMLInputElement, options: FlottformThemeOptions = {}) =>
 	(flottformBaseInputHost: FlottformFileInputHost) => {
-		// create elements (one of them being a button with "flottformFileInputHost.start()")
 		const flottformRoot =
-			options.flottformRootElement ?? document.querySelector('.flottform-root') ?? initRoot();
+			options.flottformRootElement ??
+			document.querySelector('.flottform-root') ??
+			initRoot(options.flottformRootTitle, options.flottformRootDescription);
 		const flottformItem = document.createElement('li');
 		flottformItem.setAttribute('class', `flottform-item${options.additionalItemClasses ?? ''}`);
+		inputField && setLabelForFlottformItem(inputField, options.label, flottformItem);
 		const statusInformation = document.createElement('div');
 		statusInformation.setAttribute('class', 'flottform-status-information');
-		const createChannelButton = document.createElement('button');
-		createChannelButton.setAttribute('type', 'button');
-		createChannelButton.setAttribute('class', 'flottform-button');
-		createChannelButton.innerText = 'Get a link';
+		const createChannelButton = createFlottformChannelButton(options.buttonLabel);
 		createChannelButton.addEventListener('click', () => flottformBaseInputHost.start());
-		flottformItem.appendChild(createChannelButton);
-		const refreshChannelButton = document.createElement('button');
-		refreshChannelButton.setAttribute('class', 'flottform-refresh-connection-button');
-		refreshChannelButton.setAttribute(
-			'aria-label',
-			'Click this button to refresh Flottform connection for the input field.'
-		);
-		refreshChannelButton.innerHTML = `<svg width="20" height="20" fill="currentColor" viewBox="0 0 24 24"><path d="M13.5 2c-5.288 0-9.649 3.914-10.377 9h-3.123l4 5.917 4-5.917h-2.847c.711-3.972 4.174-7 8.347-7 4.687 0 8.5 3.813 8.5 8.5s-3.813 8.5-8.5 8.5c-3.015 0-5.662-1.583-7.171-3.957l-1.2 1.775c1.916 2.536 4.948 4.182 8.371 4.182 5.797 0 10.5-4.702 10.5-10.5s-4.703-10.5-10.5-10.5z"/></svg>`;
+		const flottformStateItemsContainer = document.createElement('div');
+		flottformStateItemsContainer.setAttribute('class', 'flottform-state-items-container');
+		flottformStateItemsContainer.appendChild(createChannelButton);
+		flottformItem.appendChild(flottformStateItemsContainer);
+		const refreshChannelButton = createFlottformChannelRefreshButton();
 		refreshChannelButton.addEventListener('click', () => flottformBaseInputHost.start());
+
 		// listen to events -> change elements depending on them
 		flottformBaseInputHost.on('endpoint-created', ({ link, qrCode }) => {
 			const { createChannelQrCode, createChannelLinkWithOffer } = createLinkAndQrCode(qrCode, link);
```

Comment:
link should be an input field to be easier to copy. We should also add a `copy` button 📋

Reply:
It's done in commit [e7bcbc1](https://github.com/compose-us/flottform/pull/39/commits/e7bcbc181e1afe3a2d7195631a34dc7f5cf0c155)